models/public/midasnet/midasnet.md

```diff
@@ -0,0 +1,41 @@
+# midasnet
+
+## Use Case and High-Level Description
+
+MidasNet is a model for monocular depth estimation trained by mixing several datasets;
+as described in the following paper:
+"Towards Robust Monocular Depth Estimation: Mixing Datasets for Zero-Shot Cross-Dataset Transfer"
+<https://arxiv.org/pdf/1907.01341>
+
+The model input is a blob that consists of a single image of "1x3x384x384" in `BGR` order.
+
+The model output is an inverse depth map that is defined up to an unknown scale factor.
+
+## Example
+
+See [here](https://github.com/intel-isl/MiDaS)
+
```

Comment:
Please, add `Specification` section here (see [example](https://github.com/opencv/open_model_zoo/blob/develop/models/public/alexnet/alexnet.md#specification)).

Reply:
Ok. Is there an easy way to determine the GFLOPs and Mparams in OpenVino?

Reply:
You can use Model Analyzer tool via DL Workbench, or manually (see `<openvino_install_dir>/deployment_tools/tools/workbench/model_analyzer` directory)

Reply:
Is this also part of the dldt repo? I did not install the openvino_toolkit so far.

Reply:
Might be a silly question: I build dltl from the repo. I cannot find the model_analyzer. Do I have do enable a CMAKE flag or is it not includes in the repo?

Reply:
It was required to do this since I was earlier constrained to the master branch of dldt. This dependency is now gone with the 2020 version. Hence, I can go ahead with the toolkit...

Reply:
I am able to report the GFLOPs. However, it returns MParams: 0
Looking into the code shows: 
    def get_total_params(self) -> int:
        raise NotImplementedError

I would go ahead with just commenting on the GFLOPs

Reply:
Fine, but remove `?` sign from MParams row

Reply:
done.